channels/nz.m3u

```diff
@@ -1,4 +1,25 @@
 #EXTM3U x-tvg-url="http://i.mjh.nz/nzau/epg.xml.gz"
+
+#EXTINF:-1 tvg-id="tv.9" tvg-name="TVNZ 1" tvg-language="English" tvg-logo="https://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/TVNZ11280x1280.png" group-title="", TVNZ 1
+http://43.224.120.99:8080/tv1/tv1_1500k.m3u8
+#EXTINF:-1 tvg-id="tv.10" tvg-name="TVNZ 2" tvg-language="English" tvg-logo="https://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/TVNZ21280x1280.png" group-title="", TVNZ 2
+http://43.224.120.99:8080/tv2/tv2_1500k.m3u8
+#EXTINF:-1 tvg-id="tv.11" tvg-name="Three" tvg-language="English" tvg-logo="https://i.imgur.com/kcEdCpX.jpg" group-title="",Three
+http://mediaworks-i.akamaihd.net/hls/live/220435/3812193411001/3news_live/master.m3u8
+#EXTINF:-1 tvg-id="tv.12" tvg-name="Bravo" tvg-logo="https://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/2_FVsite_Bravo_1280x1280_v2.png" group-title="", Bravo NZ
+http://43.224.120.99:8080/four/four_1000k.m3u8
+#EXTINF:-1  tvg-id="tv.37" tvg-name="Choice TV" tvg-logo="https://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/ChannelLogoforwebsite.png" group-title="", Choice TV
+http://43.224.120.99:8080/choicetv/choicetv_1000k.m3u8
+#EXTINF:-1  tvg-id="tv.28" tvg-name="Prime" tvg-logo="https://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/Prime-2016-website.png" group-title="", Prime NZ
+http://43.224.120.99:8080/prime/prime_1000k.m3u8
+#EXTINF:-1 tvg-id="tv.28" tvg-name="Chinese TV29" tvg-logo="http://www.wtv.co.nz/wp-content/themes/wtv/images/icon/wtv28.png" group-title="",中华电视网 TV 28
+http://43.224.120.99:8080/chinesetv/chinesetv_1000k.m3u8
```

Comment:
Wrong tvg-id and tvg-name. tvg-id must be the same as the channel id in the EPG.